docs/source/distributed.rst

```diff
@@ -0,0 +1,103 @@
+.. distributed:: torch.
+
+Modes
+=====
+
+There are two modes to control the way calculations are distributed: master-worker mode and process-group mode.
+Each of them is designed to resemble interfaces and APIs familiar to PyTorch users.
+
+The process-group mode API is made to look like the MPI API.
```

Comment:
I don't think the MPI API is a great reference in a tutorial - maybe a little eplanation of the characteristics instead?